MANIFEST.in

```diff
@@ -0,0 +1 @@
+include LICENSE.md
```

Comment:
Thanks!

We should probably also add these?...

```text
global-exclude __pycache__
global-exclude *.py[co]
```

Reply:
Added them @tomchristie .